middlewares/auth.js

```diff
@@ -0,0 +1,17 @@
+const modelUser = require('../models/user')
+
+module.exports.auth = async (req, res, next) => {
+  try {
+    const tokenUser = req.get('Authorization')
+    const User = await modelUser.findOne({ token: tokenUser })
+    if (User.token != null) {
+      res.locals.email = User.email
+      res.locals.name = User.name
+      res.locals.role = User.role
+      res.locals.id = User._id
+      next()
+    } else throw new Error()
```

Comment:
I think is more readable if you have consistency with braces 
I would prefer use 
`} else {
   throw new Error()
}`

Reply:
Done